sensors/test_deps.go

```diff
@@ -102,13 +100,10 @@ func getReplaySensor(testTime string) *inject.Camera {
 	cam.ProjectorFunc = func(ctx context.Context) (transform.Projector, error) {
 		return nil, transform.NewNoIntrinsicsError("")
 	}
-	cam.PropertiesFunc = func(ctx context.Context) (camera.Properties, error) {
```

Comment:
why is this being removed? this shouldn't be removed

Reply:
I thought we are removing everything to do with properties func?

Reply:
This is a good point. This is fine

Reply:
actually, lets talk about this